sys/include/bitfield.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2015 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_bitfield bitfields of arbitrary length
```

Comment:
The current module caption clashes somewhat with the other sys modules in Doxygen which have a one or two word caption.
Suggested: `@defgroup    sys_bitfield Bitfields`